dipy/tracking/tracker.py

```diff
@@ -0,0 +1,93 @@
+from dipy.tracking.fast_tracking import generate_tractogram
+
+
+def probabilistic_directions():
+    pass
+
+
+def deterministic_directions():
+    pass
+
+
+def eudx_directions():
+    pass
+
+
+def build_stopping_criterion(mask=None, check_point=None, include=None,
+                             exclude=None):
+    sc_element = {
+        "mask": mask,
+        "wm_map": None,
+        "gm_map": None,
+        "csf_map": None,
+        "checkpoint": check_point,
+        "include": include,
+        "exclude": exclude,
```

Comment:
I know we usually do no use enums, but I would define these into an enum or an SC class; otherwise, they are not exposed/documented in the API.

Reply:
solve, added a enum `TrackerStatus`

thanks